src/hclib-deque.c

```diff
@@ -39,32 +39,38 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "hclib-internal.h"
 #include "hclib-atomics.h"
 
-void deque_init(deque_t *deq, void *init_value) {
+#if 0 // UNUSED
```

Comment:
I'd prefer actually removing all of the dead code in a single patch (or at least a set of consecutive commits), which should make it easier to track down if we need to find it again later.